package.json

```diff
@@ -77,7 +78,8 @@
     "@openzeppelin/contracts": "4.5.0",
     "@openzeppelin/contracts-upgradeable": "4.5.2",
     "@uniswap/v2-core": "1.0.1",
-    "@uniswap/v2-periphery": "1.1.0-beta.0"
+    "@uniswap/v2-periphery": "1.1.0-beta.0",
+    "lodash": "^4.17.21"
```

Comment:
This should be in devDependencies

Reply:
Looking at other projects (Open Zeppling mainly), only the code of the contracts is considered production. Tasks / deployments are tools for day to day tasks and CD, it doesn't run on production